exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.apache.drill.exec.store.parquet;
 
+import static java.lang.Thread.sleep;
```

Comment:
No need to import static `sleep` method. There are a lot of similar usages in Drill code and class usage is better for visibility.

Reply:
Changed to use Thread.sleep()